td-layout/src/lib.rs

```diff
@@ -29,18 +29,16 @@ use runtime::*;
 
 // Minimal memory size to build the runtime layout.
 #[cfg(feature = "boot-kernel")]
-pub const MIN_MEMORY_SIZE: u64 = (TD_PAYLOAD_ACPI_SIZE
-    + TD_PAYLOAD_UNACCEPTED_MEMORY_BITMAP_SIZE
-    + TD_PAYLOAD_PAGE_TABLE_SIZE
-    + TD_PAYLOAD_EVENT_LOG_SIZE
-    + TD_PAYLOAD_MAILBOX_SIZE) as u64;
+pub const MIN_MEMORY_SIZE: u64 = (ACPI_SIZE
```

Comment:
why rename???

Reply:
To align with the name defined in https://github.com/confidential-containers/td-shim/blob/main/td-layout/src/memslice.rs#L16